File: __old/c/batman.c
#include "controller.h"

int void draw(){     
    controller_left(22) 
    controller_forward(85) //linke Schulter
    controller_right(120)
    controller_up()
    controller_forward(10)
    controller_left(180)
    controller_down()
    controller_forward(65) //linker Hals
    controller_left(13)
    controller_forward(85) //linkes Gesicht
    controller_right(8)
    controller_forward(75) //linkes Ohr
    controller_right(160)
    controller_forward(60)
    controller_left(90)
    controller_forward(40) //linker Oberkopf
    controller_right(39)
    controller_forward(40) //rechter Oberkopf
    controller_left(90)
    controller_forward(60) //rechtes Ohr
    controller_right(160)
    controller_forward(75)
    controller_right(8) 
    controller_forward(85) //rechtes Gesicht
    controller_left(13)
    controller_forward(65) //rechter Hals
    controller_left(180)
    controller_up()
    controller_forward(10)
    controller_right(120)
    controller_down()
    controller_forward(85) //rechte Schulter
    controller_up()
    controller_left(180)
    controller_right(23) //direkt zur Maske
    controller_forward(38)
    //alt
    controller_forward(85) 
    //controller_right(60)
    //controller_forward(55)
    controller_left(110)
    controller_down()
    controller_forward(53) //rechte Wange
    controller_right(60)
    controller_forward(42)
    controller_right(60)
    controller_forward(53) //linke Wange
    controller_left(180)
    controller_up()
    controller_forward(20)
    controller_left(150)
    controller_down()
    controller_forward(25) //Maske links senkrecht
    controller_right(115)
    controller_forward(40) //Maske Nase links
    controller_left(50)
    controller_forward(40) //Maske Nase rechts
    controller_right(115)
    controller_forward(30) //Maske rechts senkrecht
    controller_up() //zum Auge rechts
    controller_left(170)
    //controller_forward(25)
    //controller_left(22)
    controller_forward(80)
    controller_left(115)
    controller_down()
    controller_forward(30)
    controller_left(150) // rechtes Innenauge
    controller_forward(20)
    controller_left(70) // rechtes Auge: untere Ecke
    controller_forward(20)
    controller_up()
    controller_left(125)
    controller_forward(22)
    //controller_left(90)
    //controller_forward(45) //Strecke Nase rechts
    //controller_right(85)
    //controller_forward(25)
    //controller_right(50)
    //controller_forward(20)
    //controller_right(85)
    //controller_forward(45)
    //controller_left(100)
    controller_forward(60)
    //controller_left(180)
    //controller_forward(20)
    controller_left(125)
    controller_down()
    controller_forward(20)
    controller_left(70)
    controller_forward(20)
    controller_left(150)
    controller_forward(30)
    controller_down()

    return 0;
}

File: __old/python/batman.py
import signal

try:
    from controller import Controller
    my_turtle = Controller(200.0)
except:
    from turtle import Turtle
    my_turtle = Turtle()


def draw():
    my_turtle.left(22) 
    my_turtle.down()
    my_turtle.forward(85) #linke Schulter
    my_turtle.up()
    my_turtle.right(120)
    my_turtle.forward(10)
    my_turtle.left(180)
    my_turtle.down()
    my_turtle.forward(65) #linker Hals
    my_turtle.left(13)
    my_turtle.forward(85) #linbkes Gesicht
    my_turtle.right(8)
    my_turtle.forward(75) #linkes Ohr
    my_turtle.right(160)
    my_turtle.forward(60)
    my_turtle.left(90)
    my_turtle.forward(50) #linker Oberkopf
    my_turtle.right(39)
    my_turtle.forward(50) #rechter Oberkopf
    my_turtle.left(90)
    my_turtle.forward(60) #rechtes Ohr
    my_turtle.right(160)
    my_turtle.forward(75)
    my_turtle.right(8) 
    my_turtle.forward(85) #rechtes Gesicht
    my_turtle.left(13)
    my_turtle.forward(65) #rechter Hals
    my_turtle.up()
    my_turtle.left(180)
    my_turtle.forward(10)
    my_turtle.right(120)
    my_turtle.down()
    my_turtle.forward(85) #rechte Schulter
    my_turtle.up()
    my_turtle.left(157) #direkt zur Maske
    my_turtle.forward(123)
    my_turtle.left(110)
    my_turtle.down()
    my_turtle.forward(53) #rechte Wange
    my_turtle.right(60)
    my_turtle.forward(42)
    my_turtle.right(60)
    my_turtle.forward(53) #linke Wange
    my_turtle.up()
    my_turtle.left(180)
    my_turtle.forward(20)
    my_turtle.left(150)
    my_turtle.down()
    my_turtle.forward(25) #Maske links senkrecht
    my_turtle.right(115)
    my_turtle.forward(40) #Maske Nase links
    my_turtle.left(50)
    my_turtle.forward(40) #Maske Nase rechts
    my_turtle.right(115)
    my_turtle.forward(30) #Maske rechts senkrecht
    my_turtle.up() #zum Auge rechts
    my_turtle.left(170)
    my_turtle.forward(80)
    my_turtle.left(115)
    my_turtle.down()
    my_turtle.forward(30)
    my_turtle.left(150) # rechtes Innenauge
    my_turtle.forward(20)
    my_turtle.left(70) # rechtes Auge: untere Ecke
    my_turtle.forward(20)
    my_turtle.up()
    my_turtle.left(125)
    my_turtle.forward(22)
    my_turtle.forward(60)
    my_turtle.left(125)
    my_turtle.down()
    my_turtle.forward(20)
    my_turtle.left(70)
    my_turtle.forward(20)
    my_turtle.left(150)
    my_turtle.forward(30)
    my_turtle.up()


def run():
    try:
        draw()
    except KeyboardInterrupt:
        my_turtle.up()
        raise


def exit(signum=None, frame=None):
    print('exit()')
    raise KeyboardInterrupt()


if __name__ == '__main__':
    signal.signal(signal.SIGINT, exit)
    signal.signal(signal.SIGTERM, exit)

    print('start drawing...')
    run()
    print('done')
    input()
